airbyte-ci/connectors_ci/metadata_service/orchestrator/orchestrator/sensors/catalog_sensors.py

```diff
@@ -0,0 +1,45 @@
+from typing import List
+from dagster import sensor, RunRequest, SkipReason, SensorDefinition, SensorEvaluationContext, build_resources, DefaultSensorStatus
+
+
+def generate_composite_etag_cursor(etags: List[str]):
+    return ":".join(etags)
+
+
+def catalog_updated_sensor(job, resources_def) -> SensorDefinition:
+    """
+    This sensor is responsible for checking if the latest catalog has been updated in GCS.
```

Comment:
Could you mention this sensor is polling

Reply:
Yup can do!